RELEASE_NOTES.md

```diff
@@ -9,6 +9,7 @@ Version 4.next
   - polysat
     - native word level bit-vector solving.
   - introduction of simple induction lemmas to handle a limited repertoire of induction proofs.
+  - Light quantifier elimination based on term graphs (egraphs), and corresponding Model Based Projection for arrays and ADTs. Used by Spacer and QSAT.
 
```

Comment:
It looks like qel is used to replace qe_lite in spacer, but otherwise qe-lite is exposed as usual as a tactic. 
Nowadays I would create a "simplifier" (in ast/simplifiers) that invokes qe functionality such that it can be used incrementally and as a tactic. There is a simple wrapper that takes a simplifier and creates a tactic.
It should be relatively easy because qel preserves equivalence. It can be disabled if proof production is required.

Are we really going to have both qe-lite and qel?

Reply:
In the medium term, we would like to see qe-lite be replaced with qel.

In the short term, there are still some problems that behave better with qe-lite. It is convenient to have qe-lite available for debugging and comparison until we resolve these issues as either bug fixes or expected anomalies.

It would be useful for us to switch all external users to qel to see that it is not creating any new or unexpected issues in the use-cases we have not tested for.

Do you want us to expose qel as a tactic similar to qe-lite for now?

Reply:
I can't say for sure what to do in short term: the interface to qel requires at least some wrapping to create a simplifier. The wrapping could be through a rewriter where the method for handling quantifiers is changed. So rewriter, simplifier, tactic.
Not sure it is worth writing special case code for this when it really is a quantifier rewriter which applies also to the der rewriter, and qe-lite. 

qe_light_tactic.cpp contains some qel namespace but not the calls into term graphs from what I can tell.